crates/typst-library/src/text/raw.rs

```diff
@@ -56,6 +62,38 @@ use crate::World;
 /// #raw("fn " + "main() {}", lang: "rust")
 /// ```
 ///
+/// # Styling
+/// By default, the `raw` element uses the `DejaVu Sans Mono` font, available
+/// by default in Typst, with a smaller font size of `0.8em` (that is, 80% of
```

Comment:
```suggestion
/// By default, the `raw` element uses the `DejaVu Sans Mono` font (included
/// with Typst) with a smaller font size of `{0.8em}` (that is, 80% of
```
"default" appears twice in the sentence, which is not so elegant. maybe something like this would read better.